### PR TITLE
Add media device and constraints helpers

### PR DESCRIPTION
**Pull request by havardholvik**
Moves helper functions from our PWA to here to allow re-use by our browser-sdk.

<details>
  <summary>📦 Published PR as canary version: <code>1.4.1--canary.31.6723804123.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @whereby/jslib-media@1.4.1--canary.31.6723804123.0
  # or 
  yarn add @whereby/jslib-media@1.4.1--canary.31.6723804123.0
  ```
</details>

### CODE REVIEW

File: src/webrtc/mediaConstraints.js
Comment (kevinwhereby) [inline src/webrtc/mediaConstraints.js:1]: will there be problems with existing imports from `src/MediaConstraints` if this is `mediaConstraints` now?

Comment (kevinwhereby) [reply src/webrtc/mediaConstraints.js:1]: I can't see any imports from this file anywhere, so maybe it's fine

Comment (havardholvik) [reply src/webrtc/mediaConstraints.js:1]: Yea. I actually didnt know this file existed before, and I could not see any usage of it either...

**Review by kevinwhereby — APPROVED** [review]
🚢

**Pull request merged**